README.md

```diff
@@ -1,20 +1,47 @@
-This repo contains a list of Engineering titles, roles, and associated responsibilities:
+The Mariana Tek Engineering Ladder defines the responsibilities and expectations of people working at various levels within the engineering team. It exists as a guide for engineers pursuing excellence and advancement, as well as a framework for managers to evaluate performance and identify candidates for promotion.
```

Comment:
Well phrased, and accurate to our discussions and intentions IMO!

Reply:
It's also useful as a guide for hiring.  IMO the appropriate section should be read by each interviewer before giving an interview so as to provide a framework around what questions are appropriate and what skills/qualifications we're looking for for that role.  Doing so would also help eliminate natural biases.